File: app/src/main/java/com/sagar/watchnext/network/models/tv/reviews/Review.java

package com.sagar.watchnext.network.models.tv.reviews;

import com.google.gson.annotations.SerializedName;

public class Review {

    @SerializedName("author")
    private String author;

    @SerializedName("content")
    private String content;

    @SerializedName("id")
    private String id;

    @SerializedName("url")
    private String url;

}

File: app/src/main/java/com/sagar/watchnext/network/models/movies/reviews/Review.kt
package com.sagar.watchnext.network.models.movies.reviews

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Review {

    @SerializedName("id")
    val id: String? = null

    @SerializedName("author")
    val author: String? = null

    @SerializedName("content")
    val content: String? = null

    @SerializedName("url")
    val url: String? = null
}
